PelotonEppSdk/Models/EventResponse.cs

```diff
@@ -71,18 +74,22 @@ internal class event_response : response
         {
             if (eventResponse == null) return null;
 
-            ICollection<EventItem> items = null;
-            if (eventResponse.items != null)
-                items = eventResponse.items.Select(ei => (EventItem)ei).ToList();
+            var eventState = (State)eventResponse.state;
+            EventStateEnum eventStateEnumValue = EventStateEnum.Unknown;
+            if (eventState != null)
+            {
+                eventStateEnumValue = (EventStateEnum)int.Parse(eventState.Code);
+            }
 
             return new EventResponse(eventResponse)
             {
                 Name = eventResponse.name,
+                FriendlyUrlPath = eventResponse.friendly_url_path,
                 Description = eventResponse.description,
-                StartDatetime = eventResponse.start_datetime,
-                EndDatetime = eventResponse.end_datetime,
-                State = (State)eventResponse.state,
-                Items = items,
+                StartDatetime = eventResponse.start_date,
```

Comment:
@cattiwill, yes, we are. But until that change to the EPP.API is at least in the 'test' environment, the SDK must be using 'start_date'.
Once the change is in EPP.API, I have the corresponding code changes ready in another branch for the SDK